### DISCUSSION TITLE
en/removing-outdated-content-website-seo/

### DISCUSSION THREAD

**Discussion by giscus** [idarek/dariusz.wieckiewicz.org#103] [General]
# en/removing-outdated-content-website-seo/

Free world, free thoughts, free words.

https://dariusz.wieckiewicz.org/en/removing-outdated-content-website-seo/

**Comment by Donaldx9** [comment]
Up-to-date content on a website is really important. Especially from an SEO and business point of view. I recommend this great post on the subject: https://gamerseo.com/blog/common-seo-mistakes-to-avoid-fix-them-asap/

**Reply by idarek** [reply]
Thanks. If this linked post is yours I would highly recommend to recheck things as well. I just skimmed through it and saw information about title limits... this is not relevant anymore, at least for google :)
